src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -62,6 +56,15 @@ struct
   let name () = "Polyhedra"
 end
 
+module AffEqManager =
```

Comment:
Maybe follow the format here that is also used for the other Managers, i.w. give a link to the documentation in a comment at the top.

Reply:
I added a description. Since the underlying library is also Polka, the description might seem a bit duplicated, but that should be fine I think.